packages/formstr-app/src/constants/prompts.js

```diff
@@ -0,0 +1,76 @@
+export const CREATE_FORM_TOOL_SCHEMA = {
+  type: 'object',
+  properties: {
+    title: {
+      type: 'string',
+      description: 'The main title of the form.',
+    },
+    description: {
+      type: 'string',
+      description: 'Optional: A brief description or introduction for the form.',
+    },
+    fields: {
+      type: 'array',
+      description: 'An array of objects, where each object represents a field/question in the form.',
+      items: {
+        type: 'object',
+        properties: {
+          type: {
+            type: 'string',
+            description: 'MANDATORY: The type of the form field. Must be one of the specified enum values.',
+            enum: [ 
```

Comment:
it can also be a label right? can you check the renderElement for label types and add it here?